src/client.ts

```diff
@@ -96,6 +103,48 @@ export class ZerionAPI implements iZerionAPI {
     );
     return data;
   }
+
+  async getNativeTokens(
+    chains: ChainData[],
+    params?: {
+      useStatic?: boolean;
+      supportedChains?: number[];
```

Comment:
You shouldn't need to keep this here, since you are already supplying the chains (they should already be the list you want).

Reply:
It is not the list that I want. And I will need the info to determine if I can just respond with all tokens or if I need to loop over them. I could just supply `relevantChains` as the `chains` parameter, but then I would ALWAYS have to do this iterated [lookup](https://github.com/bh2smith/zerion-sdk/pull/13/files#diff-25d66d74617fe2e23d7946bd6e3ba95640ab1b9bc8947445d604fc271c7c1f12R127-R132).

Reply:
Fair enough. Lemme just take one last look here.